packages/SwingSet/test/virtualObjects/test-virtualObjectManager.js

```diff
@@ -77,6 +84,63 @@ function zotVal(arbitrary, name, tag, count) {
   });
 }
 
+test('multifaceted virtual objects', t => {
```

Comment:
Let's make a second test here which ignores `state`, just a pair of marker facets, to exercise the problems with the cohort record not being retained. I'd have to think a while to figure out how to capture the problem in a unit test, but let's at least have *some* code which exercises that case, to give any assertions or whatever we've got a chance to fire.